dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/validation/validator/ValidationUtils.java

```diff
@@ -300,7 +301,13 @@ private static String validateImage(TrackerBundle bundle, String value) {
     FileResource fileResource = bundle.getPreheat().get(FileResource.class, value);
 
     return Constant.VALID_IMAGE_FORMATS.contains(fileResource.getFormat())
-        ? "File resource with uid '" + value + "' is not a valid image"
+        ? "File resource with uid '"
+            + value
+            + "' is using invalid image format "
+            + fileResource.getFormat()
+            + ". Valid formats are: ("
+            + StringUtils.join(Constant.VALID_IMAGE_FORMATS)
```

Comment:
AFAIK without passing a separator you'll turn `"a","b"` into `"ab"` which is probably not what we want.

Remove the `()` from the sentence.